### DISCUSSION TITLE
Inquire about the format of the competition after the Round of 32

### DISCUSSION THREAD

**Discussion by merak0514** [cyberbotics/wrestling#84] [Q&A] [answered]
Hello. I hope to know the detailed format of the competition after the Round of 32. 
1. I would like to know if the matches will continue with a group stage or transition into a knockout stage. 
2. Could you provide details on how the matchups will be determined? Like 1st-32nd, ...

Thank you.

**Comment by omichel** [comment, marked as answer]
Sure, I recently [posted](https://github.com/cyberbotics/wrestling/discussions/63#discussioncomment-5882678) all the information about it in the [main page](https://github.com/cyberbotics/wrestling/tree/main#readme) and prepared the [result page](https://github.com/cyberbotics/wrestling/blob/main/result.md) to explain how this will happen.

Please let me know if you have any question about it.
